<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultilinearFaceReconClass</class>
 <widget class="QMainWindow" name="MultilinearFaceReconClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>531</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MultilinearFaceRecon</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Target"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionFit"/>
    <addaction name="actionGenerate_Prior"/>
    <addaction name="actionStart_Kinect"/>
    <addaction name="actionReset_Tracking"/>
    <addaction name="actionBatch_Recon"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Target">
   <property name="text">
    <string>Load Target</string>
   </property>
  </action>
  <action name="actionFit">
   <property name="text">
    <string>Fit</string>
   </property>
  </action>
  <action name="actionGenerate_Prior">
   <property name="text">
    <string>Generate Prior</string>
   </property>
  </action>
  <action name="actionStart_Kinect">
   <property name="text">
    <string>Start Kinect</string>
   </property>
  </action>
  <action name="actionReset_Tracking">
   <property name="text">
    <string>Reset Tracking</string>
   </property>
  </action>
  <action name="actionBatch_Recon">
   <property name="text">
    <string>Batch Recon</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="multilinearfacerecon.qrc"/>
 </resources>
 <connections/>
</ui>
